yarn.lock

```diff
@@ -2708,7 +2708,7 @@
     cosmiconfig "^5.1.0"
     dedent "^0.7.0"
     dot-prop "^4.2.0"
-    glob-parent "^5.1.2"
+    glob-parent "^5.0.0"
```

Comment:
Entries related to `glob-parent` and `trim-newlines` will be resolved once #4867 is merged.

Reply:
@Ibrahimmaga Looks like #4867 is now merged. Can you update and resolve conflicts?

Reply:
Yes. I will resolve the conflict on this